src/design-system/components/Box/Box.tsx

```diff
@@ -284,7 +283,7 @@ export const Box = forwardRef(function Box(
 
     // We flatten the styles array in case it's passed to Animated.View.
     // This won't be needed with v2.3+ of react-native-reanimated.
```

Comment:
Yeah, we have Reanimated 2.3 already, with changes that I will soon merge we will have 2.8. I think it would be worth it to try removing it right now or just adding a TODO with a linear Ticket attached, you can assign me I will try to take care of it

Reply:
Let's test this again when the changes from reanimated are ready